pkg/vm/engine/tae/catalog/model.go

```diff
@@ -34,7 +34,7 @@ var (
 )
 
 const (
-	PhyAddrColumnName    = "PADDR"
+	PhyAddrColumnName    = "__mo_PADDR"
```

Comment:
Why is the definition inside the tae. Shouldn't this be as common as catalog?

Reply:
I find this const definition here, then use this const value.

Reply:
Move it out to common catalog.

Reply:
ok